Items/music_album.rb

```diff
@@ -0,0 +1,56 @@
+require 'colorize'
+require './Items/item'
+require './properties/genre'
+require './helper-classes/input_validation'
+require 'date'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :genre
+
+  def initialize(publish_date, title, genre, on_spotify)
+    super(publish_date, title)
+    @on_spotify = on_spotify
+    @genre = genre
+    @archived = false
+  end
+
+  def add_genre(genre)
+    genre.albums.push(self) unless genre.albums.include?(self)
+  end
+
+  def self.list_all_albums(albums)
+    puts 'No music albums added, please add an album by using the list of options.' if albums.empty?
+    albums.each do |album|
+      puts "Title: #{album.title}  Release: #{album.publish_date}\n\n
```

Comment:
I would suggest displaying the index of the albums as this can improve the user experience :+1:

Reply:
@aron-helu Thank you so much for pointing this out mate! :smile: I have already implemented it :heavy_check_mark: